apps/cms/lib/partial/paragraph/code_embed.ex

```diff
@@ -0,0 +1,30 @@
+defmodule CMS.Partial.Paragraph.CodeEmbed do
+  @moduledoc """
+  Represents a CodeEmbed Paragraph in the CMS.
+  Body is raw HTML from CMS, and is not scrubbed or rewritten.
+  """
+
+  alias CMS.Partial.Paragraph.ColumnMultiHeader
+  alias Phoenix.HTML
+
+  import CMS.Helpers, only: [field_value: 2, parse_paragraphs: 3]
+
+  defstruct header: nil,
+            right_rail: false,
+            body: HTML.raw("")
+
+  @type t :: %__MODULE__{
+          header: ColumnMultiHeader.t() | nil,
+          right_rail: boolean,
+          body: HTML.safe()
```

Comment:
@arkadyan should this be set to `HTML.raw()`?

Reply:
I'm not sure I follow the question, could you clarify it, please? It should be whatever the type of the data is, but I'm not sure that's what you are asking. I'm also not finding what `HTML.raw()` is.

Reply:
Nevermind -- this is a typespec, and there is only a HTML.safe() type.